proxy/cc-proxy.spec-template

```diff
@@ -18,6 +18,7 @@ License  : Apache-2.0 GPL-2.0
 
 Requires: cc-proxy-bin
 Requires: cc-proxy-config
+Requires: clear-containers-selinux
```

Comment:
Isn't there a way to make this dynamic?

I meant, clear-containers-selinux is required **only** if selinux is installed in the system.

If we can add a conditional there would be great.